packages/legend-query-builder/src/stores/filter/QueryBuilderFilterState.ts

```diff
@@ -164,90 +320,84 @@ export class FilterConditionState implements Hashable {
     }
   }
 
-  changeProperty(propertyExpression: AbstractPropertyExpression): void {
```

Comment:
This function is no longer used since we no longer support DND a property on top of the existing filter property to replace it.